### PR TITLE
[Solved #65] create add player effect

### PR DESCRIPTION
**Pull request by supunTE**
Solved #65 
- Create an effect for `add-player`
- Add netlify redirects file

### CODE REVIEW

**Comment by supunTE** [comment]
Working on unit tests

File: libs/store/src/lib/effects/player.ts [hunk 1]
```diff
@@ -132,6 +133,79 @@ export const joinPlayer = (
...
+    return;
+  }
+
+  dispatch.game.setTournamentState({
+    tournamentId,
+    tournamentState,
+  });
+
+  // Add the new player.
+  dispatch.game.addPlayerReducer({
```
Comment (dalanad) [inline libs/store/src/lib/effects/player.ts:197]: why does `addPlayerReducer` has reducer as a suffix and `setTournamentState` doesn't ?

**Review by dalanad — CHANGES_REQUESTED** [review]

File: libs/store/src/lib/effects/player.ts (hunk 1 above)
Comment (supunTE) [reply libs/store/src/lib/effects/player.ts:197]: `setTournamentState` is an effect, not a reducer. In our application effects will call necessary reducers to update the store. (Sometimes other effects also). Reducers to fundamental updates on store like add, remove, update while effects process data and pass to effect.

 Here AddPlayer effect will call addPlayer reducer with processed data.

**Review by dalanad — APPROVED** [review]

**Comment by supunTE** [comment]
Solved #65

**Pull request merged**